mk_verificator/tests/system/migration/config.yaml

```diff
@@ -0,0 +1,11 @@
+---
+admin_username: admin
+admin_password: 1111
+admin_project_id: 1111
+url: ""
+url_v3: ""
+endpoint_type: internalURL
+image_ref: b994881e-773a-4227-94cb-1082015a0930
```

Comment:
I suggest to move endpoint_type, image_ref, flavor_ref, network_ref to global config